source/common/http/conn_manager_config.h

```diff
@@ -466,6 +467,10 @@ class ConnectionManagerConfig {
    * @return LocalReply configuration which supplies mapping for local reply generated by Envoy.
    */
   virtual const LocalReply::LocalReply& localReply() const PURE;
+
+  virtual PathTransformer* forwardingPathTransformer() const { return nullptr; }
```

Comment:
We should keep the `ConnectionManagerConfig` interface purely abstract. I.e. only PURE methods. This will make your CL slightly bigger as you will need add implementation for the new PURE methods there.
We should also return `PathTransformer` by reference and not pointer. In this case you will need to use ReturnByRef when setting up `EXPECT_CALL` expectations.

Reply:
Thanks, will fix.